### ISSUE TITLE
Invalid Reset Token for password resets

### ISSUE THREAD

**Issue by waynegraham** [clirdlf/participatory_voting#22]
When a user attempts to reset a password, an email is generated (through sendgrid) with a valid looking reset token (e.g. `users/password/edit?reset_password_token=JNCsASpiGsX8oKBi985c`). However, when the user changes the email, they are notified that the **Reset password token is invalid**.

## Work Around
You can manually generate a password reset token with the following:

```
heroku run rails console

raw, hashed = Devise.token_generator.generate(User, :reset_password_token)

user = User.find_by(email: 'email@address.com')
user.reset_password_token = hashed
user.reset_password_sent_at = Time.now.utc
user.save

puts "https://voting.diglib.org/users/password/edit?reset_password_token=#{raw}"
```

**Comment by waynegraham** [comment]
Immediate workaround for passwords:

```
heroku run rails console
email_borked = 'email@address.com'
u = User.where(email: email_borked).first
u.email = email_borked + ".borked"
u.save
```

**Comment by waynegraham** [comment]
closed in 8d913ff4ccad7ebafa1038c205d77b66e8040336
